website/docs/add-secure-apps/flows-stages/flow/context/index.mdx

```diff
@@ -203,3 +208,22 @@ If _Show matched user_ is disabled, this key will be set to the user identifier
 [Set this key](../../../../customize/policies/expression/managing_flow_context_keys.md) in an Expression Policy to override [Redirect stage](../../stages/redirect/index.md) to force it to redirect to a certain URL or flow. This is useful when a flow requires that the redirection target be decided dynamically.
 
 Use the format `ak-flow://{slug}` to use the Redirect stage in Flow mode. Any other format will result in the Redirect stage running in Static mode.
+
+#### Mutual TLS Stage
+
+##### `certificate` (dictionary):ak-version[2025.6]
+
+This key is set by the Mutual TLS Stage during enrollment and contains data about the certificate supplied by the browser.
```

Comment:
Ideally can we say the name of the key, and not "this key" in the first sentence?

Reply:
"This key" refers to `certificate` in the header above this text, I think that's clear enough? Cause we do the same thing for all the sections above

Reply:
It's still a messy and unprofessional start to a sentence, sorry. :-) Stating the name again is also easier for the reader... but as you wish.

I'd propose "The `certificate` key is set by the Mutual..."